### DISCUSSION TITLE
[Done v1.0.0 Release Checklist

### DISCUSSION THREAD

**Discussion by lokka30** [ArcanePlugins/Treasury#125] [General]
- [x] Test Treasury
- [x] Complete #57 
- [x] Complete #123 
- [x] Receive @MrNemo64's approval on #115 
- [x] Write Wiki page on the basics of using the EconomyProvider. Especially narrow down on how the Subscriber is used. 

Once released, and a few economy providers are out, I need to test the migration command.

**Comment by MrNemo64** [comment]
So... What now? Everything is checked up

**Reply by lokka30** [reply]
Check the first page of the Spigot resources section ;)

**Reply by MrNemo64** [reply]
What are we working after this? I suppose y'all want to work on your own providers and consumers to promote the api but the economy adapter should be the priority and developing the other APIs is something that we can start thinking about

**Reply by lokka30** [reply]
Ivan and I have decided to scrap the adapter idea entirely, we rather either full or no Treasury support

I'll also start working on my economy provider tomorrow but it'll take a while to complete since I'm at the keyboard, if it's not perfect then it's not going out the door 😛 

Once I've finished my provider I'll PR Treasury support into plugins that are interested in it

**Reply by MrNemo64** [reply]
> Ivan and I have decided to scrap the adapter idea entirely, we rather either full or no Treasury support

What were the reasons behind it?

**Reply by MrIvanPlays** [reply]
> > Ivan and I have decided to scrap the adapter idea entirely, we rather either full or no Treasury support
> 
> What were the reasons behind it?

that would make developers lazy, which would mean they won't use the full potential of the treasury api. we appreciate anyone that spreads the word - treasury's been released and developers can start adding support for it.

**Reply by lokka30** [reply]
I also realized that economy providers (like mine) will probably have Vault and Treasury support at the same time. so if economy providers will do it, then there is no need for anything else apart from each plugin adding direct Treasury integration
